spec/views/catalog/sort_and_per_page.html.erb_spec.rb

```diff
@@ -15,13 +15,18 @@
     allow(view).to receive(:blacklight_config).and_return(CatalogController.blacklight_config)
   end
 
-
-
   it 'appears on page without error' do
     render
     page = Capybara::Node::Simple.new(rendered)
     expect(page).to have_selector('span.page_entries', count: 1)
     expect(rendered).to include("<strong>4</strong> - <strong>6</strong> of <strong>20</strong>")
   end
+  
+  it 'does not use a directional arrow for relevance sorting' do
```

Comment:
I prefer we just test for what it does do. There are lots of things it does not do that we are not testing for.